tests/dotnet/UnitTests/ProjectTest.cs

```diff
@@ -1666,5 +1663,40 @@ public void StrippedRuntimeIdentifiers (ApplePlatform platform, string runtimeId
 			Assert.That (symbols, Does.Contain ("_xamarin_release_managed_ref"), "_xamarin_release_managed_ref");
 		}
 
+		[Test]
+		[TestCase (ApplePlatform.iOS, "ios-arm64;")]
+		[TestCase (ApplePlatform.MacOSX, "osx-arm64;osx-x64")]
+		[TestCase (ApplePlatform.MacCatalyst, "maccatalyst-x64")]
+		[TestCase (ApplePlatform.TVOS, "tvos-arm64;")]
+		public void SourcelinkTest (ApplePlatform platform, string runtimeIdentifiers)
+		{
+			// Sourcelink uses the latest commit and tests to see if
+			// it is valid which will not pass until the commit has
+			// been merged in and actually exists on github.
+
+			if (!IsInCI || IsPullRequest)
+				Assert.Ignore ("This test is disabled for local runs and Pull Requests.");
+
+			var project = "MySimpleApp";
+
+			var project_path = GetProjectPath (project, runtimeIdentifiers: runtimeIdentifiers, platform: platform, out var appPath);
+			Clean (project_path);
+			var properties = GetDefaultProperties (runtimeIdentifiers);
+			var rv = DotNet.AssertBuild (project_path, properties);
+
+			var pdbFile = Directory
+				.GetFiles (Path.GetDirectoryName (project_path)!, "*.pdb", SearchOption.AllDirectories)
+				.FirstOrDefault ();
+
+			Assert.NotNull (pdbFile, "No PDB file found");
+
+			using Process install = new ();
+			install.StartWithArgs ("tool install sourcelink");
```

Comment:
I think this would be better as a `InstallTool` helper method on the DotNet class, something like the existing `InstallWorkload` helper method:

https://github.com/xamarin/xamarin-macios/blob/e96bf83be3c40b1512ebd1c92c6081d8f8af392e/tests/common/DotNet.cs#L116-L138

Reply:
ah missed this pr, thanks for the suggestion- made the updates!